### DISCUSSION TITLE
Form factors

### DISCUSSION THREAD

**Discussion by iandoug** [binarybottle/engram-es#26] [Ideas]
Hi

Is ISO the standard in Spain?

Am struggling to get ISO to perform as well as ANSI (with basically the same layout), the distances to the shifts and Enter work against the format ...

**Comment by Lobo-Feroz** [comment]
Hello there,

Yes, ISO is the default here. 

One of the proposed goals of a theoretically optimized spanish kb was to potentially have also a corresponding ANSI version. But I think the default one should still be ISO, which is what everyone has here.
